README.md

```diff
@@ -3,6 +3,9 @@
 [![Slack chat][slack-image]][slack-url]
 [![Build Status][circleci-image]][circleci-url]
 
+**OTEPs have been moved to the [Specification](https://github.com/open-telemetry/opentelemetry-specification/tree/main/oteps/)
+repository. This repository has been preserved for reference purposes. Please otherwise refer to the Specification.**
```

Comment:
```suggestion
**:warning: OTEPs have been moved to the [Specification](https://github.com/open-telemetry/opentelemetry-specification/tree/main/oteps/)
repository. This repository has been preserved for reference purposes. Please otherwise refer to the Specification.**
```